pymc_marketing/mmm/base.py

```diff
@@ -1126,7 +1127,17 @@ def plot_grouped_contribution_breakdown_over_time(
         area_params = dict(stacked=True, ax=ax)
         if area_kwargs is not None:
             area_params.update(area_kwargs)
-        all_contributions_over_time.plot.area(**area_params)
+        try:
+            all_contributions_over_time.plot.area(**area_params)
```

Comment:
Thoughts of check on frame values instead of try except?

Reply:
The cheks are done in the plot.area function itself and can have many edge cases. For now lets direct the responsibility to matplotlib 🙏